native/Avalonia.Native/src/OSX/Screens.mm

```diff
@@ -43,14 +66,44 @@ virtual HRESULT GetScreen (int index, AvnScreen* ret) override
             
             ret->Scaling = 1;
```

Comment:
Does anybody remember why we set scaling to 1? Instead of using an actual value (more likely between 2 and 3 for most main displays).

Reply:
> Our Screen API reports layout scaling

Is it used anywhere like that? It's also not intuitive.
Normally people would use Window.DesktopScaling for that.

Reply:
Other implementations like qtbase and chromium just return dpi scaling, not layout.

Reply:
Will keep it as is for now